ffi/examples/read-table/schema.h

```diff
@@ -78,7 +85,24 @@ void print_list(SchemaBuilder* builder, uintptr_t list_id, int indent, int paren
     }
     SchemaItem* item = &list->list[i];
     char* prefix = is_last ? "└" : "├";
-    printf("%s─ %s: %s\n", prefix, item->name, item->type);
+    printf("%s─ %s: %s", prefix, item->name, item->type);
+    if (strcmp(item->type, "array") == 0) {
```

Comment:
previously we were trying to shove this info into the item name. but we were using the item itself's nullability, which is wrong. we want to see if the child item is nullable or not, and then print that info out.

also made it more clear that we're talking about _contains_ null vs. if the item itself can be null.